### PR TITLE
Bug 1436892 / Bug 1436891 - Enable sections on dashboard for specific groups

### PR DESCRIPTION
**Pull request by MichaelKohler**
* Enables "Reps without Mentors" section for Council and Peers members - previously Admin only
* Enables "Reps without profile" section for Council and Onboarding members - previously Admin only

### CODE REVIEW

**Comment by MichaelKohler** [comment]
@akatsoulas r?

File: remo/dashboard/templates/dashboard_reps.jinja [hunk 1]
```diff
@@ -1052,8 +1052,7 @@
 {% endif %}
 <!-- end planning requests block -->
 
-<!-- Admin block -->
-{% if is_admin is defined %}
+<!-- Strategy block -->
```
Comment (akatsoulas) [inline remo/dashboard/templates/dashboard_reps.jinja]: By removing the `if` completely, all the users will be able to see an empty `Administration` heading. Probably you need to augment the if statement by adding all the users that should have access to it.

**Comment by akatsoulas** [comment]
Can you please squash the three commits into one?

File: remo/dashboard/templates/dashboard_reps.jinja (hunk 1 above)
Comment (MichaelKohler) [reply remo/dashboard/templates/dashboard_reps.jinja]: Oh, you're completely right, missed that. Will do

**Comment by MichaelKohler** [comment]
@akatsoulas  Done!

**Pull request merged**